dev_guide/application_ephemeral_storage_sizing.adoc

```diff
@@ -0,0 +1,143 @@
+[[dev-guide-application-ephemeral-storage-sizing]]
+= Application ephemeral storage sizing
```

Comment:
Is "application ephemeral storage" another type? Or is this the same as "runtime ephemeral storage"?

Reply:
Same thing.  Again, which term shall I use throughout?

Reply:
I would use application since that it the title of this topic.

Reply:
Thus will keep as is.